packages/psss/src/views/Tabs/AlertsTabView.js

```diff
@@ -63,28 +57,32 @@ export const AlertsTabView = React.memo(() => {
   );
 
   return (
-    <Container>
-      <Main>
-        <AlertsTable period={period} onRowClick={handleTableRowClick} />
-        <AlertsPanel
-          countryCode={rowData.organisationUnit}
-          period={rowData.period}
-          syndromeName={rowData.syndromeName}
-          isOpen={isPanelOpen}
-          handleClose={handlePanelClose}
-        />
-      </Main>
-      <Sidebar>
-        <Card variant="outlined">
-          <CardHeader color="primary" title="Current Week" label={DatePicker} />
-          <CardContent>
-            <Typography variant="h4">{`Week ${getWeekNumberByPeriod(period)}`}</Typography>
-            <DateSubtitle variant="subtitle2" gutterBottom>
-              {getDisplayDatesByPeriod(period)}
-            </DateSubtitle>
-          </CardContent>
-        </Card>
-      </Sidebar>
-    </Container>
+    <div style={{ position: 'relative' }}>
+      {alertsCategory === 'outbreaks' && (
+        <ComingSoon text="The Outbreaks page will show archived Alerts and Outbreaks." />
+      )}
+      <Container>
+        <Main>
+          <AlertsPanelProvider>
+            {/* TODO Outbreak and Alert components could be merged when outbreaks are implemented */}
+            {alertsCategory === 'outbreaks' ? <OutbreaksTable /> : <AlertsTable period={period} />}
+            {alertsCategory === 'outbreaks' ? <OutbreaksPanel /> : <AlertsPanel />}
+          </AlertsPanelProvider>
+        </Main>
+        {alertsCategory !== 'outbreaks' && (
+          <Sidebar>
+            <Card variant="outlined">
+              <CardHeader color="primary" title="Selected Week" label={DatePicker} />
```

Comment:
I know this is from a previous change but I missed it first time around. My understanding of the card in the side of the page was that it just showed the current iso week and then all the alerts were shown in the table. Is there any reason to not show all the alerts? One would imagine the list will never be very long at one time and navigating to alerts in the past doesn't seem like a realistic use case. Perhaps it's a question for a PO or designer?

Reply:
I believe I got this wrong... I see what you mean. Let's also discuss about this